source/guides/non-denizen/resource-packs.md

```diff
@@ -0,0 +1,248 @@
+```json
+{
+  "entity.player.defence.level": {
+    "sounds": [
+      {
+        "name": "custom/defence_levelup0"
+      },
+      {
+        "name": "custom/defence_levelup1"
+      }
+    ],
+    "subtitle": "Excited Trumpet Noises"
+  }
+}
+```
+
+The first key is the name for the command; in this example, `entity.player.defence.level`
+The only data object within the command we need to specify are `sounds`. Optionally, you can specify the subtitle that displays if subtitles are enabled in-game. If you place multiple sounds within the `"sounds":[]` array, the sound will randomize between them based on their weight.
+
+For each file, you will need the data: `"name":"FILEPATH/FILENAME"`, excluding the file's extension.
+Optionally, you can manually adjust the following valid properties of the sound:
+1) volume - The volume the sound will be played as. 
+    - Default is `1.0`; Valid volume ranges from `0.0` to `1.0`; where `1.0` is the loudest it may be played at.
+    - The Volume value accepts higher values using Denizen's PlaySound, however not by increasing the volume. It increases the audible distance the sound may be heard from. Ie; volume 5 can be heard from five chunks away.
+2) pitch - The pitch the sound plays at, altered from it's original `.ogg` form.
+    - Default is `1.0`; Valid pitches range from `0.0` to `2.0`; where `0.0` is high-pitched and `1.0` will be low-pitched.
+3) weight - The chance that this sopund will be selected as opposed to randomly.
+    - Default is 0; Only accepts valid integers, not adjustable within Denizen.
+4) stream - Determines if the sound should be streamed from it's file.
+    - Default is false
+    - Recommended to set this value as `true` if the sound is longer than two seconds to avoid lag.
+    - Use this sparingly; it's not optimal to specify everything true.
+    - This is used with all music disks.
+5) preload - Determines if the sound should be loaded when loading the pack, as opposed to when the player plays the sound. 
+    - Default is false
+    - Used for ambient noises.
+6) attenuation_distance - Determines the reduction rate based on distance.
+    - Default is `1.0`
+    - Used by portals, beacons, conduits
+7) type - determines if a pre-defined event fires this sound.
+    - Default is `sound`
+    - used for things like being under-water, in a cave, near a beacon, near a beehive.
+
+### Inside The Blockstates Folder Directory
+Directory: `.minecraft\resourcepacks\MyResourcePack\assets\minecraft\blockstates\`
+To modify each individual block-state of an item, you must specify each individual blockstate.
+Additional blockstates cannot be specified. When specifying blockstate models, the relative folder directs to the `Models` directory, located at `\assets\minecraft\models\`
+Adjusting these are not traditional, therefor not covered further in this guide.
```

Comment:
`therefor` <-- typo.
`are` <-- `is`\*?
`traditional` <-- `normal`\*? or `common`\*?